academic-services/knowledge-exploration-service/get-started-create-api-instances.md

```diff
@@ -5,24 +5,79 @@ ms.topic: tutorial
 ms.date: 01/03/2020
 ---
 
-# Create MAKES API instances using MAKES command line tool
+# Deploying a basic MAKES instance to Azure from your Subscription
+
+ Step-by-step guide for deploying a basic MAKES instance from a Microsoft Academic Knowledge Exploration Service (MAKES) release. For more information on obtaining a MAKES release, visit [Get Microsoft Academic Knowledge Exploration Service on Azure storage](get-started-setup-provisioning.md)
 
 ## Prerequisites
 
-- MAKES release
-- MAKES indexes/grammar
+- Microsoft Academic Knowledge Exploration Service (MAKES) subscription
 - Azure subscription
 
-## Deploy MAKES API hosting resources
+## Verify the current release
```

Comment:
For step-by-steps we should use numbered lists as it helps users keep track of where they are.

Reply:
Sorry, I actually intended this for the bulleted lists inside the sections

---
In reply to: [370777199](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r370777199) [](ancestors = 370777199)